core/src/util/mapProjection.h

```diff
@@ -137,6 +136,8 @@ class MercatorProjection : public MapProjection {
      * Following define the boundry covered by this mercator projection
      */
     int m_TileSize;
+    double m_Res;
```

Comment:
So far it's only used for internal calculations within MercatorProjection. If it will be certainly shared by all MapProjections, then can be moved it back.